test/gtest/common/gtest.h

```diff
@@ -7465,13 +7465,13 @@ struct IndexSequence {
 // Double the IndexSequence, and one if plus_one is true.
 template <bool plus_one, typename T, size_t sizeofT>
 struct DoubleSequence;
-template <size_t... I, size_t sizeofT>
-struct DoubleSequence<true, IndexSequence<I...>, sizeofT> {
-  using type = IndexSequence<I..., (sizeofT + I)..., 2 * sizeofT>;
+template <size_t... M, size_t sizeofT>
```

Comment:
instead of all these changes in gtest.h i suggest we redefine imaginary I:
in common/test.h, first include complex.h then redefine I then include gtest.h
```
include <complex.h>
#undef I
#define _Imag _Imaginary_I
#include <gtest.h>
```

then you can use "_Imag" instead of "I" in tests. have a look here : https://pubs.opengroup.org/onlinepubs/007904975/basedefs/complex.h.html

Reply:
done